src/BizHawk.Emulation.DiscSystem/DiscIdentifier.cs

```diff
@@ -185,6 +190,9 @@ public DiscType DetectDiscType()
 			if (DetectWii())
 				return DiscType.Wii;
 
+			if (DetectISO9660()) return DiscType.DOS;

```

Comment:
This should be moved much lower, after other ISO checks, DetectISO9660 is what the below [iso.Parse](https://github.com/TASEmulators/BizHawk/blob/6dccb759cba8c00d3cdbbb7d397649eadab0ed15/ExternalProjects/iso-parser/ISOVolumeDescriptor.cs#L220) does (along with CDI so it's not strictly ISO9660 anyways)